### ISSUE TITLE
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=20491#c3

### ISSUE THREAD

**Issue by huili2** [cisco/openh264#3282]
@xiaotiansf  @xiaotianshi2 could you please check the memory violation issue in above link?
Thanks in advance.

**Comment by xiaotiansf** [comment]
@huili2 I sent you e-mail on April 27 regarding this issue. I had tried everything in as many as times with its test case on different platform by using oss-fuzz test source code. Unfortunately, the bug just can't be reproduced. Do you happen to be able to reproduce the bug?

**Comment by huili2** [comment]
yes. I used Mac and directly decoded the data provided in that bug, and it can be re-produced by USE_ASAN=Yes.

**Comment by xiaotianshi2** [comment]
I see and I'll try. Just forgot how to enable USE_ASAN=Yes? Thx.

**Comment by huili2** [comment]
in Mac, "make USE_ASAN=Yes ..."

**Comment by xiaotianshi2** [comment]
@huili2 Just let you know I did not forget the issue and it is just I still try to find time to fix it. Hopefully next week.

**Comment by huili2** [comment]
OK. Thanks for the information.
